.github/workflows/conformance.yml

```
@@ -0,0 +1,292 @@
+            encryption-type: ipsec 
+            tunnel-mode: vxlan
+            nodeport: false
+
+          - name: 'v1.15 w no kpr & bpf-masq'
+            cilium-version: "v1.15.1"
+            talos-version: 'v1.6.2'
+            kube-proxy: true
+            kube-proxy-replacement: "false"
+            socketlb: true
+            bpf-masquerade: true
+            ipam-mode: 'kubernetes'
+            ipv4: true
+            ipv6: false
+            encryption-enabled: false
+            tunnel-mode: vxlan
+            nodeport: false
+
+          - name: 'v1.15 clusterpool ipam mode'
+            cilium-version: "v1.15.1"
+            talos-version: 'v1.6.2'
+            kube-proxy: false
+            kube-proxy-replacement: "true"
+            socketlb: false 
+            bpf-masquerade: true
+            ipam-mode: 'cluster-pool'
+            ipv4: true
+            ipv6: false
+            encryption-enabled: false
+            encryption-type: ipsec
+            tunnel-mode: vxlan
+            nodeport: true
+
+          - name: 'v1.15 w geneve tunnel'
+            cilium-version: "v1.15.1"
+            talos-version: 'v1.6.2'
+            kube-proxy: false
+            kube-proxy-replacement: "true"
+            socketlb: false 
+            bpf-masquerade: true
+            ipam-mode: 'kubernetes'
+            ipv4: true
+            ipv6: false
+            encryption-enabled: false
+            encryption-type: ipsec
+            tunnel-mode: geneve 
+            nodeport: true
+
+          # --- Cilium v1.14 --- 
+          
+          - name: 'v1.14'
+            cilium-version: "1.14.7"
+            talos-version: 'v1.6.2'
+            kube-proxy: false 
+            kube-proxy-replacement: "true"
+            socketlb: false 
+            bpf-masquerade: true
+            ipam-mode: 'kubernetes'
+            ipv4: true
+            ipv6: false
+            encryption-enabled: false
+            encryption-type: ipsec
+            tunnel-mode: vxlan
+            nodeport: true
+
+          - name: 'v1.14 w ipsec'
+            talos-version: 'v1.6.2'
+            cilium-version: "1.14.7"
+            kube-proxy: true 
+            kube-proxy-replacement: "false"
+            socketlb: true
+            bpf-masquerade: false
+            ipam-mode: 'kubernetes'
+            ipv4: true
+            ipv6: false
+            encryption-enabled: true
+            encryption-type: ipsec 
+            tunnel-mode: vxlan
+            nodeport: false
+
+          - name: 'v1.14 w wireguard'
+            cilium-version: "1.14.7"
+            talos-version: 'v1.6.2'
+            kube-proxy: false
+            kube-proxy-replacement: "true"
+            socketlb: false 
+            bpf-masquerade: true
+            ipam-mode: 'kubernetes'
+            ipv4: true
+            ipv6: false
+            encryption-enabled: false
+            encryption-type: wireguard
+            tunnel-mode: vxlan
+            nodeport: false
```

```suggestion
            nodeport: true
```

Would be required for `bpf-masquerade: true`, no? https://docs.cilium.io/en/stable/network/concepts/masquerading/#ebpf-based